docs/developers/v2/getting-started.md

```diff
@@ -9,9 +9,10 @@ Before you start, you will need to download and install the following requiremen
 - [ganache](https://trufflesuite.com/ganache)
 - Text editor (VS Code, Sublime Text, vim, others)
 
-If you are focusing on a strategy:
+If you are focusing on a strategy, you have the choice of brownie and foundry template repos:
 
-- [brownie-strategy-mix](https://github.com/yearn/brownie-strategy-mix) repository. For vault integrations
+- [brownie-strategy-mix](https://github.com/yearn/brownie-strategy-mix) repository. 
+- [foundry-strategy-mix](https://github.com/storming0x/foundry_strategy_mix) repository. (See Readme.MD for setup.)
```

Comment:
Can you remove  `(See Readme.MD for setup.)`? So we keep all the items consistent